epictrack-web/src/components/work/WorkForm/index.tsx

```diff
@@ -133,12 +134,27 @@ export default function WorkForm({ ...props }) {
     ctx.setTitle(ctx.item ? work?.title : "Create Work");
   }, [ctx.title, ctx.item]);
 
-  const title = watch("title");
-
   React.useEffect(() => {
     reset(ctx.item ?? defaultWork);
   }, [ctx.item]);
 
+  useEffect(() => {
+    // Find the object where name is 'None'
+    const noneFederalInvolvement = federalInvolvements.find(
+      ({ name }) => name === "None"
+    );
+    const noneSubstitutionAct = substitutionActs.find(
+      ({ name }) => name === "None"
+    );
+
+    if (
+      noneSubstitutionAct &&
+      Number(federal_involvement_id) === noneFederalInvolvement?.id
+    ) {
+      setValue("substitution_act_id", noneSubstitutionAct?.id);
+    }
+  }, [federal_involvement_id, setValue]);

```

Comment:
Do we need this setValue in the dependency list. I am not sure. Could you please make sure it works without this dependency.

Reply:
sure

Reply:
it works just tested it